main/src/ca/uwaterloo/flix/language/ast/OccurrenceAst.scala

```diff
@@ -45,7 +45,7 @@ object OccurrenceAst {
 
   object Expr {
 
-    case class Constant(cst: Ast.Constant, tpe: MonoType, loc: SourceLocation) extends OccurrenceAst.Expr {
+    case class Constant(cst: shared.Constant, tpe: MonoType, loc: SourceLocation) extends OccurrenceAst.Expr {
```

Comment:
Just import directly

Reply:
The name clashes though

Reply:
Ah, I think Expr.Constant should be Expr.Cst

Reply:
Solving the clash.